kernel/aster-nix/src/net/socket/ip/datagram/unbound.rs

```diff
@@ -36,4 +37,9 @@ impl UnboundDatagram {
 
         Ok(BoundDatagram::new(bound_socket))
     }
+
+    pub(super) fn reset_io_events(&self, pollee: &Pollee) {
+        pollee.del_events(IoEvents::IN);
+        pollee.add_events(IoEvents::OUT);
+    }
 }
```

Comment:
This method should only reset `self.pollee`, instead any pollee passed. I suggest refactor the method as
```rust
fn reset_io_events(&self) {
    self.pollee.reset_events();
    self.pollee.add_events(IoEvents::OUT)
}
```